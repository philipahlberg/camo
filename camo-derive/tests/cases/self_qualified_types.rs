use std::ops::Not;
use camo_derive::Camo;

#[derive(Camo)]
struct Foo {
    f: <i32 as Not>::Output,
}

fn main() {}
