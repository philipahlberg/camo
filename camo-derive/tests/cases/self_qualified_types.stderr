error: `camo` does not support self-qualified types in paths
 --> tests/cases/self_qualified_types.rs:6:9
  |
6 |     f: <i32 as Not>::Output,
  |         ^^^
